module-1/src/Components/MovieTile/MovieTile.tsx

```diff
@@ -43,6 +94,20 @@ export const MovieTile = ({ movie }: MovieTileProps) => {
           </p>
         ))}
       </div>
+      <Dialog openDialog={openEditDialog}>
```

Comment:
It is a bad idea to put dialogs here. Now every movie title creates its own dialog. There should be only one dialog of each type (create, delete, anything else unique) on the page and each movie will populate the dialog with appropriate data.

Reply:
fixed